2024-08-09 12:05:15,316 INFO    StreamThr :527183 [internal.py:wandb_internal():86] W&B internal server running at pid: 527183, started at: 2024-08-09 12:05:15.316296
2024-08-09 12:05:15,317 DEBUG   HandlerThread:527183 [handler.py:handle_request():146] handle_request: status
2024-08-09 12:05:15,318 INFO    WriterThread:527183 [datastore.py:open_for_write():85] open: /home/rwkv/JL/GMM/llama/wandb/run-20240809_120515-kc1lvzvh/run-kc1lvzvh.wandb
2024-08-09 12:05:15,318 DEBUG   SenderThread:527183 [sender.py:send():382] send: header
2024-08-09 12:05:15,354 DEBUG   SenderThread:527183 [sender.py:send():382] send: run
2024-08-09 12:05:15,525 INFO    SenderThread:527183 [dir_watcher.py:__init__():211] watching files in: /home/rwkv/JL/GMM/llama/wandb/run-20240809_120515-kc1lvzvh/files
2024-08-09 12:05:15,525 INFO    SenderThread:527183 [sender.py:_start_run_threads():1136] run started: kc1lvzvh with start time 1723176315.316214
2024-08-09 12:05:15,534 DEBUG   HandlerThread:527183 [handler.py:handle_request():146] handle_request: check_version
2024-08-09 12:05:15,534 DEBUG   SenderThread:527183 [sender.py:send_request():409] send_request: check_version
2024-08-09 12:05:15,637 DEBUG   HandlerThread:527183 [handler.py:handle_request():146] handle_request: run_start
2024-08-09 12:05:15,646 DEBUG   HandlerThread:527183 [system_info.py:__init__():32] System info init
2024-08-09 12:05:15,646 DEBUG   HandlerThread:527183 [system_info.py:__init__():47] System info init done
2024-08-09 12:05:15,646 INFO    HandlerThread:527183 [system_monitor.py:start():194] Starting system monitor
2024-08-09 12:05:15,646 INFO    SystemMonitor:527183 [system_monitor.py:_start():158] Starting system asset monitoring threads
2024-08-09 12:05:15,646 INFO    HandlerThread:527183 [system_monitor.py:probe():214] Collecting system info
2024-08-09 12:05:15,646 INFO    SystemMonitor:527183 [interfaces.py:start():190] Started cpu monitoring
2024-08-09 12:05:15,647 INFO    SystemMonitor:527183 [interfaces.py:start():190] Started disk monitoring
2024-08-09 12:05:15,647 INFO    SystemMonitor:527183 [interfaces.py:start():190] Started gpu monitoring
2024-08-09 12:05:15,648 INFO    SystemMonitor:527183 [interfaces.py:start():190] Started memory monitoring
2024-08-09 12:05:15,648 INFO    SystemMonitor:527183 [interfaces.py:start():190] Started network monitoring
2024-08-09 12:05:15,658 DEBUG   HandlerThread:527183 [system_info.py:probe():196] Probing system
2024-08-09 12:05:15,659 DEBUG   HandlerThread:527183 [system_info.py:_probe_git():181] Probing git
2024-08-09 12:05:15,661 DEBUG   HandlerThread:527183 [system_info.py:_probe_git():189] Probing git done
2024-08-09 12:05:15,661 DEBUG   HandlerThread:527183 [system_info.py:probe():244] Probing system done
2024-08-09 12:05:15,661 DEBUG   HandlerThread:527183 [system_monitor.py:probe():223] {'os': 'Linux-6.5.0-44-generic-x86_64-with-glibc2.35', 'python': '3.10.13', 'heartbeatAt': '2024-08-09T04:05:15.658909', 'startedAt': '2024-08-09T04:05:15.310709', 'docker': None, 'cuda': None, 'args': ('--local_rank=0', '--deepspeed', 'configs/ds_config_zero2_no_offload.json', '--model_name_or_path', '/home/rwkv/JL/model/Gemma-2B', '--data_path', '/home/rwkv/JL/model/MetaMathQA', '--dataset_field', 'query', 'response', '--dataset_split', 'train[:100000]', '--output_dir', '/home/rwkv/JL/out_model/llama-test', '--num_train_epochs', '1', '--model_max_length', '512', '--per_device_train_batch_size', '1', '--gradient_accumulation_steps', '32', '--save_strategy', 'steps', '--save_steps', '4000', '--save_total_limit', '2', '--learning_rate', '2e-5', '--weight_decay', '0.', '--warmup_ratio', '0.', '--logging_steps', '1', '--lr_scheduler_type', 'cosine', '--report_to', 'wandb', '--merge', 'True', '--run_name', 'gbmm'), 'state': 'running', 'program': '/home/rwkv/JL/GMM/llama/gmm.py', 'codePathLocal': 'gmm.py', 'codePath': 'llama/gmm.py', 'git': {'remote': 'https://github.com/JL-er/GMM.git', 'commit': '74dce7f004b5b16bfd24c8ca22e737aee6ff0769'}, 'email': '997529190@qq.com', 'root': '/home/rwkv/JL/GMM', 'host': 'rwkv-Z790M-AORUS-ELITE-AX', 'username': 'rwkv', 'executable': '/home/rwkv/miniconda3/bin/python', 'cpu_count': 20, 'cpu_count_logical': 28, 'cpu_freq': {'current': 1961.4462142857142, 'min': 800.0, 'max': 5000.0}, 'cpu_freq_per_core': [{'current': 1553.088, 'min': 800.0, 'max': 5500.0}, {'current': 800.0, 'min': 800.0, 'max': 5500.0}, {'current': 2573.214, 'min': 800.0, 'max': 5500.0}, {'current': 800.0, 'min': 800.0, 'max': 5500.0}, {'current': 1623.631, 'min': 800.0, 'max': 5500.0}, {'current': 800.0, 'min': 800.0, 'max': 5500.0}, {'current': 1743.603, 'min': 800.0, 'max': 5500.0}, {'current': 800.0, 'min': 800.0, 'max': 5500.0}, {'current': 1764.596, 'min': 800.0, 'max': 5600.0}, {'current': 800.0, 'min': 800.0, 'max': 5600.0}, {'current': 1499.06, 'min': 800.0, 'max': 5600.0}, {'current': 800.0, 'min': 800.0, 'max': 5600.0}, {'current': 1706.88, 'min': 800.0, 'max': 5500.0}, {'current': 800.0, 'min': 800.0, 'max': 5500.0}, {'current': 800.0, 'min': 800.0, 'max': 5500.0}, {'current': 1749.118, 'min': 800.0, 'max': 5500.0}, {'current': 1647.368, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 1519.709, 'min': 800.0, 'max': 4300.0}, {'current': 1451.523, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}], 'disk': {'/': {'total': 916.2753829956055, 'used': 481.3310012817383}}, 'gpu': 'NVIDIA GeForce RTX 4090', 'gpu_count': 1, 'gpu_devices': [{'name': 'NVIDIA GeForce RTX 4090', 'memory_total': 25757220864}], 'memory': {'total': 62.55397415161133}}
2024-08-09 12:05:15,661 INFO    HandlerThread:527183 [system_monitor.py:probe():224] Finished collecting system info
2024-08-09 12:05:15,661 INFO    HandlerThread:527183 [system_monitor.py:probe():227] Publishing system info
2024-08-09 12:05:15,661 DEBUG   HandlerThread:527183 [system_info.py:_save_pip():52] Saving list of pip packages installed into the current environment
2024-08-09 12:05:15,662 DEBUG   HandlerThread:527183 [system_info.py:_save_pip():68] Saving pip packages done
2024-08-09 12:05:15,662 DEBUG   HandlerThread:527183 [system_info.py:_save_conda():75] Saving list of conda packages installed into the current environment
2024-08-09 12:05:16,525 INFO    Thread-12 :527183 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240809_120515-kc1lvzvh/files/requirements.txt
2024-08-09 12:05:16,526 INFO    Thread-12 :527183 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240809_120515-kc1lvzvh/files/conda-environment.yaml
2024-08-09 12:05:17,323 DEBUG   HandlerThread:527183 [system_info.py:_save_conda():87] Saving conda packages done
2024-08-09 12:05:17,323 DEBUG   HandlerThread:527183 [system_info.py:_save_code():90] Saving code
2024-08-09 12:05:17,325 DEBUG   HandlerThread:527183 [system_info.py:_save_code():111] Saving code done
2024-08-09 12:05:17,325 DEBUG   HandlerThread:527183 [system_info.py:_save_patches():128] Saving git patches
2024-08-09 12:05:17,364 DEBUG   HandlerThread:527183 [system_info.py:_save_patches():170] Saving git patches done
2024-08-09 12:05:17,364 INFO    HandlerThread:527183 [system_monitor.py:probe():229] Finished publishing system info
2024-08-09 12:05:17,367 DEBUG   SenderThread:527183 [sender.py:send():382] send: files
2024-08-09 12:05:17,367 INFO    SenderThread:527183 [sender.py:_save_file():1392] saving file wandb-metadata.json with policy now
2024-08-09 12:05:17,367 INFO    SenderThread:527183 [sender.py:_save_file():1392] saving file code/llama/gmm.py with policy now
2024-08-09 12:05:17,367 INFO    SenderThread:527183 [sender.py:_save_file():1392] saving file diff.patch with policy now
2024-08-09 12:05:17,375 DEBUG   HandlerThread:527183 [handler.py:handle_request():146] handle_request: stop_status
2024-08-09 12:05:17,375 DEBUG   SenderThread:527183 [sender.py:send_request():409] send_request: stop_status
2024-08-09 12:05:17,381 DEBUG   HandlerThread:527183 [handler.py:handle_request():146] handle_request: internal_messages
2024-08-09 12:05:17,525 INFO    Thread-12 :527183 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/rwkv/JL/GMM/llama/wandb/run-20240809_120515-kc1lvzvh/files/conda-environment.yaml
2024-08-09 12:05:17,525 INFO    Thread-12 :527183 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240809_120515-kc1lvzvh/files/wandb-metadata.json
2024-08-09 12:05:17,525 INFO    Thread-12 :527183 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240809_120515-kc1lvzvh/files/diff.patch
2024-08-09 12:05:17,525 INFO    Thread-12 :527183 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240809_120515-kc1lvzvh/files/code/llama/gmm.py
2024-08-09 12:05:17,525 INFO    Thread-12 :527183 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240809_120515-kc1lvzvh/files/code/llama
2024-08-09 12:05:17,526 INFO    Thread-12 :527183 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240809_120515-kc1lvzvh/files/code
2024-08-09 12:05:17,557 ERROR   wandb-upload_0:527183 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/kc1lvzvh/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040517Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=d4f0d45d4b8502d62a750f3f455c6e1257aceaa2c63007ec491f68d1e75ef450: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/kc1lvzvh/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040517Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=d4f0d45d4b8502d62a750f3f455c6e1257aceaa2c63007ec491f68d1e75ef450
2024-08-09 12:05:17,557 ERROR   wandb-upload_0:527183 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '5250', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-09 12:05:17,558 ERROR   wandb-upload_0:527183 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/kc1lvzvh/wandb-metadata.json</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/kc1lvzvh/wandb-metadata.json</Resource><Region>wandb-local</Region><RequestId>17E9F3BB2317FC8B</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-09 12:05:17,562 DEBUG   SenderThread:527183 [sender.py:send():382] send: telemetry
2024-08-09 12:05:17,562 DEBUG   SenderThread:527183 [sender.py:send():382] send: config
2024-08-09 12:05:17,562 DEBUG   SenderThread:527183 [sender.py:send():382] send: metric
2024-08-09 12:05:17,562 DEBUG   SenderThread:527183 [sender.py:send():382] send: telemetry
2024-08-09 12:05:17,562 DEBUG   SenderThread:527183 [sender.py:send():382] send: metric
2024-08-09 12:05:17,562 WARNING SenderThread:527183 [sender.py:send_metric():1343] Seen metric with glob (shouldn't happen)
2024-08-09 12:05:17,562 DEBUG   SenderThread:527183 [sender.py:send():382] send: telemetry
2024-08-09 12:05:17,562 DEBUG   SenderThread:527183 [sender.py:send():382] send: telemetry
2024-08-09 12:05:17,562 DEBUG   SenderThread:527183 [sender.py:send():382] send: config
2024-08-09 12:05:17,565 DEBUG   SenderThread:527183 [sender.py:send():382] send: exit
2024-08-09 12:05:17,565 INFO    SenderThread:527183 [sender.py:send_exit():589] handling exit code: 1
2024-08-09 12:05:17,565 INFO    SenderThread:527183 [sender.py:send_exit():591] handling runtime: 1
2024-08-09 12:05:17,566 INFO    SenderThread:527183 [sender.py:_save_file():1392] saving file wandb-summary.json with policy end
2024-08-09 12:05:17,566 INFO    SenderThread:527183 [sender.py:send_exit():597] send defer
2024-08-09 12:05:17,566 DEBUG   HandlerThread:527183 [handler.py:handle_request():146] handle_request: defer
2024-08-09 12:05:17,566 INFO    HandlerThread:527183 [handler.py:handle_request_defer():172] handle defer: 0
2024-08-09 12:05:17,566 DEBUG   SenderThread:527183 [sender.py:send_request():409] send_request: defer
2024-08-09 12:05:17,566 INFO    SenderThread:527183 [sender.py:send_request_defer():613] handle sender defer: 0
2024-08-09 12:05:17,566 INFO    SenderThread:527183 [sender.py:transition_state():617] send defer: 1
2024-08-09 12:05:17,566 DEBUG   HandlerThread:527183 [handler.py:handle_request():146] handle_request: defer
2024-08-09 12:05:17,566 INFO    HandlerThread:527183 [handler.py:handle_request_defer():172] handle defer: 1
2024-08-09 12:05:17,566 DEBUG   SenderThread:527183 [sender.py:send_request():409] send_request: defer
2024-08-09 12:05:17,566 INFO    SenderThread:527183 [sender.py:send_request_defer():613] handle sender defer: 1
2024-08-09 12:05:17,566 INFO    SenderThread:527183 [sender.py:transition_state():617] send defer: 2
2024-08-09 12:05:17,567 DEBUG   HandlerThread:527183 [handler.py:handle_request():146] handle_request: defer
2024-08-09 12:05:17,567 INFO    HandlerThread:527183 [handler.py:handle_request_defer():172] handle defer: 2
2024-08-09 12:05:17,567 INFO    HandlerThread:527183 [system_monitor.py:finish():203] Stopping system monitor
2024-08-09 12:05:17,567 DEBUG   SystemMonitor:527183 [system_monitor.py:_start():172] Starting system metrics aggregation loop
2024-08-09 12:05:17,567 INFO    HandlerThread:527183 [interfaces.py:finish():202] Joined cpu monitor
2024-08-09 12:05:17,567 DEBUG   SystemMonitor:527183 [system_monitor.py:_start():179] Finished system metrics aggregation loop
2024-08-09 12:05:17,567 INFO    HandlerThread:527183 [interfaces.py:finish():202] Joined disk monitor
2024-08-09 12:05:17,567 DEBUG   SystemMonitor:527183 [system_monitor.py:_start():183] Publishing last batch of metrics
2024-08-09 12:05:17,593 INFO    HandlerThread:527183 [interfaces.py:finish():202] Joined gpu monitor
2024-08-09 12:05:17,593 INFO    HandlerThread:527183 [interfaces.py:finish():202] Joined memory monitor
2024-08-09 12:05:17,593 INFO    HandlerThread:527183 [interfaces.py:finish():202] Joined network monitor
2024-08-09 12:05:17,594 DEBUG   SenderThread:527183 [sender.py:send_request():409] send_request: defer
2024-08-09 12:05:17,594 INFO    SenderThread:527183 [sender.py:send_request_defer():613] handle sender defer: 2
2024-08-09 12:05:17,594 INFO    SenderThread:527183 [sender.py:transition_state():617] send defer: 3
2024-08-09 12:05:17,594 DEBUG   SenderThread:527183 [sender.py:send():382] send: stats
2024-08-09 12:05:17,594 DEBUG   HandlerThread:527183 [handler.py:handle_request():146] handle_request: defer
2024-08-09 12:05:17,594 INFO    HandlerThread:527183 [handler.py:handle_request_defer():172] handle defer: 3
2024-08-09 12:05:17,594 DEBUG   SenderThread:527183 [sender.py:send_request():409] send_request: defer
2024-08-09 12:05:17,594 INFO    SenderThread:527183 [sender.py:send_request_defer():613] handle sender defer: 3
2024-08-09 12:05:17,594 INFO    SenderThread:527183 [sender.py:transition_state():617] send defer: 4
2024-08-09 12:05:17,594 DEBUG   HandlerThread:527183 [handler.py:handle_request():146] handle_request: defer
2024-08-09 12:05:17,594 INFO    HandlerThread:527183 [handler.py:handle_request_defer():172] handle defer: 4
2024-08-09 12:05:17,594 DEBUG   SenderThread:527183 [sender.py:send_request():409] send_request: defer
2024-08-09 12:05:17,594 INFO    SenderThread:527183 [sender.py:send_request_defer():613] handle sender defer: 4
2024-08-09 12:05:17,594 INFO    SenderThread:527183 [sender.py:transition_state():617] send defer: 5
2024-08-09 12:05:17,594 DEBUG   HandlerThread:527183 [handler.py:handle_request():146] handle_request: defer
2024-08-09 12:05:17,594 INFO    HandlerThread:527183 [handler.py:handle_request_defer():172] handle defer: 5
2024-08-09 12:05:17,594 DEBUG   SenderThread:527183 [sender.py:send():382] send: summary
2024-08-09 12:05:17,594 INFO    SenderThread:527183 [sender.py:_save_file():1392] saving file wandb-summary.json with policy end
2024-08-09 12:05:17,594 DEBUG   SenderThread:527183 [sender.py:send_request():409] send_request: defer
2024-08-09 12:05:17,594 INFO    SenderThread:527183 [sender.py:send_request_defer():613] handle sender defer: 5
2024-08-09 12:05:17,594 INFO    SenderThread:527183 [sender.py:transition_state():617] send defer: 6
2024-08-09 12:05:17,594 DEBUG   HandlerThread:527183 [handler.py:handle_request():146] handle_request: defer
2024-08-09 12:05:17,594 INFO    HandlerThread:527183 [handler.py:handle_request_defer():172] handle defer: 6
2024-08-09 12:05:17,594 DEBUG   SenderThread:527183 [sender.py:send_request():409] send_request: defer
2024-08-09 12:05:17,594 INFO    SenderThread:527183 [sender.py:send_request_defer():613] handle sender defer: 6
2024-08-09 12:05:17,596 DEBUG   HandlerThread:527183 [handler.py:handle_request():146] handle_request: status_report
2024-08-09 12:05:17,673 INFO    SenderThread:527183 [sender.py:transition_state():617] send defer: 7
2024-08-09 12:05:17,673 DEBUG   HandlerThread:527183 [handler.py:handle_request():146] handle_request: defer
2024-08-09 12:05:17,673 INFO    HandlerThread:527183 [handler.py:handle_request_defer():172] handle defer: 7
2024-08-09 12:05:17,673 DEBUG   SenderThread:527183 [sender.py:send_request():409] send_request: defer
2024-08-09 12:05:17,673 INFO    SenderThread:527183 [sender.py:send_request_defer():613] handle sender defer: 7
2024-08-09 12:05:17,693 ERROR   wandb-upload_1:527183 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/kc1lvzvh/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040517Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=e3f48fc3f1005b7250b059ea08238d9d1c0245a1ac1ad9980ea2c8708559a31c: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/kc1lvzvh/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040517Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=e3f48fc3f1005b7250b059ea08238d9d1c0245a1ac1ad9980ea2c8708559a31c
2024-08-09 12:05:17,693 ERROR   wandb-upload_1:527183 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '14196', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-09 12:05:17,693 ERROR   wandb-upload_1:527183 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/kc1lvzvh/code/llama/gmm.py</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/kc1lvzvh/code/llama/gmm.py</Resource><Region>wandb-local</Region><RequestId>17E9F3BB27D5020C</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-09 12:05:17,941 ERROR   wandb-upload_2:527183 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/kc1lvzvh/diff.patch?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040517Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=89f14b4d3106910c6d563976ed77151506afdab52d67c7f7a996d3977b6a9a17: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/kc1lvzvh/diff.patch?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040517Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=89f14b4d3106910c6d563976ed77151506afdab52d67c7f7a996d3977b6a9a17
2024-08-09 12:05:17,941 ERROR   wandb-upload_2:527183 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '294205', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-09 12:05:17,941 ERROR   wandb-upload_2:527183 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/kc1lvzvh/diff.patch</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/kc1lvzvh/diff.patch</Resource><Region>wandb-local</Region><RequestId>17E9F3BB275605AC</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-09 12:05:18,526 INFO    Thread-12 :527183 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/rwkv/JL/GMM/llama/wandb/run-20240809_120515-kc1lvzvh/files/config.yaml
2024-08-09 12:05:18,527 INFO    Thread-12 :527183 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240809_120515-kc1lvzvh/files/output.log
2024-08-09 12:05:18,527 INFO    Thread-12 :527183 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240809_120515-kc1lvzvh/files/wandb-summary.json
2024-08-09 12:05:18,566 DEBUG   HandlerThread:527183 [handler.py:handle_request():146] handle_request: poll_exit
2024-08-09 12:05:18,840 ERROR   wandb-upload_0:527183 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/kc1lvzvh/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040517Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=d4f0d45d4b8502d62a750f3f455c6e1257aceaa2c63007ec491f68d1e75ef450: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/kc1lvzvh/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040517Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=d4f0d45d4b8502d62a750f3f455c6e1257aceaa2c63007ec491f68d1e75ef450
2024-08-09 12:05:18,840 ERROR   wandb-upload_0:527183 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '5250', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-09 12:05:18,840 ERROR   wandb-upload_0:527183 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/kc1lvzvh/wandb-metadata.json</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/kc1lvzvh/wandb-metadata.json</Resource><Region>wandb-local</Region><RequestId>17E9F3BB6FB1475C</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-09 12:05:18,980 ERROR   wandb-upload_1:527183 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/kc1lvzvh/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040517Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=e3f48fc3f1005b7250b059ea08238d9d1c0245a1ac1ad9980ea2c8708559a31c: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/kc1lvzvh/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040517Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=e3f48fc3f1005b7250b059ea08238d9d1c0245a1ac1ad9980ea2c8708559a31c
2024-08-09 12:05:18,980 ERROR   wandb-upload_1:527183 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '14196', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-09 12:05:18,981 ERROR   wandb-upload_1:527183 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/kc1lvzvh/code/llama/gmm.py</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/kc1lvzvh/code/llama/gmm.py</Resource><Region>wandb-local</Region><RequestId>17E9F3BB7806E356</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-09 12:05:18,981 INFO    wandb-upload_1:527183 [retry.py:__call__():172] Retry attempt failed:
Traceback (most recent call last):
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 2759, in upload_file
    response.raise_for_status()
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/kc1lvzvh/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040517Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=e3f48fc3f1005b7250b059ea08238d9d1c0245a1ac1ad9980ea2c8708559a31c

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/lib/retry.py", line 131, in __call__
    result = self._call_fn(*args, **kwargs)
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 2785, in upload_file
    raise _e.with_traceback(sys.exc_info()[2])
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 2759, in upload_file
    response.raise_for_status()
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
wandb.sdk.lib.retry.TransientError: None
2024-08-09 12:05:19,258 ERROR   wandb-upload_2:527183 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/kc1lvzvh/diff.patch?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040517Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=89f14b4d3106910c6d563976ed77151506afdab52d67c7f7a996d3977b6a9a17: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/kc1lvzvh/diff.patch?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040517Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=89f14b4d3106910c6d563976ed77151506afdab52d67c7f7a996d3977b6a9a17
2024-08-09 12:05:19,258 ERROR   wandb-upload_2:527183 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '294205', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-09 12:05:19,258 ERROR   wandb-upload_2:527183 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/kc1lvzvh/diff.patch</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/kc1lvzvh/diff.patch</Resource><Region>wandb-local</Region><RequestId>17E9F3BB840C22B5</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-09 12:05:19,577 INFO    SenderThread:527183 [sender.py:transition_state():617] send defer: 8
2024-08-09 12:05:19,577 DEBUG   SenderThread:527183 [sender.py:send_request():409] send_request: poll_exit
2024-08-09 12:05:19,577 DEBUG   HandlerThread:527183 [handler.py:handle_request():146] handle_request: defer
2024-08-09 12:05:19,577 INFO    HandlerThread:527183 [handler.py:handle_request_defer():172] handle defer: 8
2024-08-09 12:05:19,577 DEBUG   SenderThread:527183 [sender.py:send_request():409] send_request: defer
2024-08-09 12:05:19,577 INFO    SenderThread:527183 [sender.py:send_request_defer():613] handle sender defer: 8
2024-08-09 12:05:19,577 INFO    SenderThread:527183 [job_builder.py:build():298] Attempting to build job artifact
2024-08-09 12:05:19,578 INFO    SenderThread:527183 [job_builder.py:_get_source_type():428] is repo sourced job
2024-08-09 12:05:19,593 INFO    SenderThread:527183 [job_builder.py:build():404] adding wandb-job metadata file
2024-08-09 12:05:19,605 INFO    SenderThread:527183 [sender.py:transition_state():617] send defer: 9
2024-08-09 12:05:19,606 DEBUG   SenderThread:527183 [sender.py:send():382] send: artifact
2024-08-09 12:05:19,606 DEBUG   HandlerThread:527183 [handler.py:handle_request():146] handle_request: defer
2024-08-09 12:05:19,608 INFO    HandlerThread:527183 [handler.py:handle_request_defer():172] handle defer: 9
2024-08-09 12:05:20,004 ERROR   wandb-upload_3:527183 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/wandb_artifacts/42/540/c2df8f0a6cb4e05226b857b6cdc53f4d?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040519Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=161ee69e2e6c5e620bd7fe41cb0f3d56c03e89c5fd96c34cdc728897a14f9c75: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/wandb_artifacts/42/540/c2df8f0a6cb4e05226b857b6cdc53f4d?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040519Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=161ee69e2e6c5e620bd7fe41cb0f3d56c03e89c5fd96c34cdc728897a14f9c75
2024-08-09 12:05:20,004 ERROR   wandb-upload_3:527183 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'x-amz-meta-md5': 'wt+PCmy04FImuFe2zcU/TQ==', 'Content-MD5': 'wt+PCmy04FImuFe2zcU/TQ==', 'Content-Type': 'text/plain; charset=utf-8', 'Content-Length': '5025', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-09 12:05:20,004 ERROR   wandb-upload_3:527183 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>wandb_artifacts/42/540/c2df8f0a6cb4e05226b857b6cdc53f4d</Key><BucketName>local-files</BucketName><Resource>/local-files/wandb_artifacts/42/540/c2df8f0a6cb4e05226b857b6cdc53f4d</Resource><Region>wandb-local</Region><RequestId>17E9F3BBB4E81998</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-09 12:05:20,118 ERROR   wandb-upload_4:527183 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/wandb_artifacts/42/540/8f642e05e18a200ebb54b5ecf2b46c8f?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040519Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=7314ab73bf6efea60d0b531338262d129c41805920572033a03e8d1422de929c: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/wandb_artifacts/42/540/8f642e05e18a200ebb54b5ecf2b46c8f?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040519Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=7314ab73bf6efea60d0b531338262d129c41805920572033a03e8d1422de929c
2024-08-09 12:05:20,118 ERROR   wandb-upload_4:527183 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'x-amz-meta-md5': 'j2QuBeGKIA67VLXs8rRsjw==', 'Content-MD5': 'j2QuBeGKIA67VLXs8rRsjw==', 'Content-Type': 'application/json', 'Content-Length': '25171', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-09 12:05:20,118 ERROR   wandb-upload_4:527183 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>wandb_artifacts/42/540/8f642e05e18a200ebb54b5ecf2b46c8f</Key><BucketName>local-files</BucketName><Resource>/local-files/wandb_artifacts/42/540/8f642e05e18a200ebb54b5ecf2b46c8f</Resource><Region>wandb-local</Region><RequestId>17E9F3BBB70EE3B4</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-09 12:05:20,527 INFO    Thread-12 :527183 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/rwkv/JL/GMM/llama/wandb/run-20240809_120515-kc1lvzvh/files/output.log
2024-08-09 12:05:21,145 ERROR   wandb-upload_3:527183 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/wandb_artifacts/42/540/c2df8f0a6cb4e05226b857b6cdc53f4d?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040519Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=161ee69e2e6c5e620bd7fe41cb0f3d56c03e89c5fd96c34cdc728897a14f9c75: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/wandb_artifacts/42/540/c2df8f0a6cb4e05226b857b6cdc53f4d?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040519Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=161ee69e2e6c5e620bd7fe41cb0f3d56c03e89c5fd96c34cdc728897a14f9c75
2024-08-09 12:05:21,145 ERROR   wandb-upload_3:527183 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'x-amz-meta-md5': 'wt+PCmy04FImuFe2zcU/TQ==', 'Content-MD5': 'wt+PCmy04FImuFe2zcU/TQ==', 'Content-Type': 'text/plain; charset=utf-8', 'Content-Length': '5025', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-09 12:05:21,145 ERROR   wandb-upload_3:527183 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>wandb_artifacts/42/540/c2df8f0a6cb4e05226b857b6cdc53f4d</Key><BucketName>local-files</BucketName><Resource>/local-files/wandb_artifacts/42/540/c2df8f0a6cb4e05226b857b6cdc53f4d</Resource><Region>wandb-local</Region><RequestId>17E9F3BBF8DB3E9F</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-09 12:05:21,210 ERROR   wandb-upload_0:527183 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/kc1lvzvh/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040517Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=d4f0d45d4b8502d62a750f3f455c6e1257aceaa2c63007ec491f68d1e75ef450: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/kc1lvzvh/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040517Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=d4f0d45d4b8502d62a750f3f455c6e1257aceaa2c63007ec491f68d1e75ef450
2024-08-09 12:05:21,210 ERROR   wandb-upload_0:527183 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '5250', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-09 12:05:21,210 ERROR   wandb-upload_0:527183 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/kc1lvzvh/wandb-metadata.json</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/kc1lvzvh/wandb-metadata.json</Resource><Region>wandb-local</Region><RequestId>17E9F3BBFD04FBFC</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-09 12:05:21,268 ERROR   wandb-upload_4:527183 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/wandb_artifacts/42/540/8f642e05e18a200ebb54b5ecf2b46c8f?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040519Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=7314ab73bf6efea60d0b531338262d129c41805920572033a03e8d1422de929c: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/wandb_artifacts/42/540/8f642e05e18a200ebb54b5ecf2b46c8f?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040519Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=7314ab73bf6efea60d0b531338262d129c41805920572033a03e8d1422de929c
2024-08-09 12:05:21,268 ERROR   wandb-upload_4:527183 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'x-amz-meta-md5': 'j2QuBeGKIA67VLXs8rRsjw==', 'Content-MD5': 'j2QuBeGKIA67VLXs8rRsjw==', 'Content-Type': 'application/json', 'Content-Length': '25171', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-09 12:05:21,268 ERROR   wandb-upload_4:527183 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>wandb_artifacts/42/540/8f642e05e18a200ebb54b5ecf2b46c8f</Key><BucketName>local-files</BucketName><Resource>/local-files/wandb_artifacts/42/540/8f642e05e18a200ebb54b5ecf2b46c8f</Resource><Region>wandb-local</Region><RequestId>17E9F3BBFE484D4B</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-09 12:05:21,312 ERROR   wandb-upload_1:527183 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/kc1lvzvh/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040517Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=e3f48fc3f1005b7250b059ea08238d9d1c0245a1ac1ad9980ea2c8708559a31c: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/kc1lvzvh/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040517Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=e3f48fc3f1005b7250b059ea08238d9d1c0245a1ac1ad9980ea2c8708559a31c
2024-08-09 12:05:21,312 ERROR   wandb-upload_1:527183 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '14196', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-09 12:05:21,312 ERROR   wandb-upload_1:527183 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/kc1lvzvh/code/llama/gmm.py</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/kc1lvzvh/code/llama/gmm.py</Resource><Region>wandb-local</Region><RequestId>17E9F3BC030618A6</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-09 12:05:21,746 ERROR   wandb-upload_2:527183 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/kc1lvzvh/diff.patch?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040517Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=89f14b4d3106910c6d563976ed77151506afdab52d67c7f7a996d3977b6a9a17: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/kc1lvzvh/diff.patch?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040517Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=89f14b4d3106910c6d563976ed77151506afdab52d67c7f7a996d3977b6a9a17
2024-08-09 12:05:21,746 ERROR   wandb-upload_2:527183 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '294205', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-09 12:05:21,746 ERROR   wandb-upload_2:527183 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/kc1lvzvh/diff.patch</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/kc1lvzvh/diff.patch</Resource><Region>wandb-local</Region><RequestId>17E9F3BC1BC345EC</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-09 12:05:23,504 ERROR   wandb-upload_3:527183 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/wandb_artifacts/42/540/c2df8f0a6cb4e05226b857b6cdc53f4d?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040519Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=161ee69e2e6c5e620bd7fe41cb0f3d56c03e89c5fd96c34cdc728897a14f9c75: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/wandb_artifacts/42/540/c2df8f0a6cb4e05226b857b6cdc53f4d?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040519Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=161ee69e2e6c5e620bd7fe41cb0f3d56c03e89c5fd96c34cdc728897a14f9c75
2024-08-09 12:05:23,504 ERROR   wandb-upload_3:527183 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'x-amz-meta-md5': 'wt+PCmy04FImuFe2zcU/TQ==', 'Content-MD5': 'wt+PCmy04FImuFe2zcU/TQ==', 'Content-Type': 'text/plain; charset=utf-8', 'Content-Length': '5025', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-09 12:05:23,504 ERROR   wandb-upload_3:527183 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>wandb_artifacts/42/540/c2df8f0a6cb4e05226b857b6cdc53f4d</Key><BucketName>local-files</BucketName><Resource>/local-files/wandb_artifacts/42/540/c2df8f0a6cb4e05226b857b6cdc53f4d</Resource><Region>wandb-local</Region><RequestId>17E9F3BC855D240C</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-09 12:05:23,504 INFO    wandb-upload_3:527183 [retry.py:__call__():172] Retry attempt failed:
Traceback (most recent call last):
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 2759, in upload_file
    response.raise_for_status()
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/wandb_artifacts/42/540/c2df8f0a6cb4e05226b857b6cdc53f4d?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040519Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=161ee69e2e6c5e620bd7fe41cb0f3d56c03e89c5fd96c34cdc728897a14f9c75

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/lib/retry.py", line 131, in __call__
    result = self._call_fn(*args, **kwargs)
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 2785, in upload_file
    raise _e.with_traceback(sys.exc_info()[2])
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 2759, in upload_file
    response.raise_for_status()
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
wandb.sdk.lib.retry.TransientError: None
2024-08-09 12:05:23,521 ERROR   wandb-upload_4:527183 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/wandb_artifacts/42/540/8f642e05e18a200ebb54b5ecf2b46c8f?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040519Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=7314ab73bf6efea60d0b531338262d129c41805920572033a03e8d1422de929c: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/wandb_artifacts/42/540/8f642e05e18a200ebb54b5ecf2b46c8f?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040519Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=7314ab73bf6efea60d0b531338262d129c41805920572033a03e8d1422de929c
2024-08-09 12:05:23,521 ERROR   wandb-upload_4:527183 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'x-amz-meta-md5': 'j2QuBeGKIA67VLXs8rRsjw==', 'Content-MD5': 'j2QuBeGKIA67VLXs8rRsjw==', 'Content-Type': 'application/json', 'Content-Length': '25171', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-09 12:05:23,521 ERROR   wandb-upload_4:527183 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>wandb_artifacts/42/540/8f642e05e18a200ebb54b5ecf2b46c8f</Key><BucketName>local-files</BucketName><Resource>/local-files/wandb_artifacts/42/540/8f642e05e18a200ebb54b5ecf2b46c8f</Resource><Region>wandb-local</Region><RequestId>17E9F3BC84DAAC72</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-09 12:05:23,521 INFO    wandb-upload_4:527183 [retry.py:__call__():172] Retry attempt failed:
Traceback (most recent call last):
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 2759, in upload_file
    response.raise_for_status()
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/wandb_artifacts/42/540/8f642e05e18a200ebb54b5ecf2b46c8f?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040519Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=7314ab73bf6efea60d0b531338262d129c41805920572033a03e8d1422de929c

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/lib/retry.py", line 131, in __call__
    result = self._call_fn(*args, **kwargs)
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 2785, in upload_file
    raise _e.with_traceback(sys.exc_info()[2])
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 2759, in upload_file
    response.raise_for_status()
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
wandb.sdk.lib.retry.TransientError: None
2024-08-09 12:05:23,577 DEBUG   HandlerThread:527183 [handler.py:handle_request():146] handle_request: keepalive
2024-08-09 12:05:25,656 ERROR   wandb-upload_1:527183 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/kc1lvzvh/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040517Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=e3f48fc3f1005b7250b059ea08238d9d1c0245a1ac1ad9980ea2c8708559a31c: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/kc1lvzvh/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040517Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=e3f48fc3f1005b7250b059ea08238d9d1c0245a1ac1ad9980ea2c8708559a31c
2024-08-09 12:05:25,657 ERROR   wandb-upload_1:527183 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '14196', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-09 12:05:25,657 ERROR   wandb-upload_1:527183 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/kc1lvzvh/code/llama/gmm.py</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/kc1lvzvh/code/llama/gmm.py</Resource><Region>wandb-local</Region><RequestId>17E9F3BD05FCF28E</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-09 12:05:25,933 ERROR   wandb-upload_2:527183 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/kc1lvzvh/diff.patch?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040517Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=89f14b4d3106910c6d563976ed77151506afdab52d67c7f7a996d3977b6a9a17: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/kc1lvzvh/diff.patch?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040517Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=89f14b4d3106910c6d563976ed77151506afdab52d67c7f7a996d3977b6a9a17
2024-08-09 12:05:25,933 ERROR   wandb-upload_2:527183 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '294205', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-09 12:05:25,933 ERROR   wandb-upload_2:527183 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/kc1lvzvh/diff.patch</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/kc1lvzvh/diff.patch</Resource><Region>wandb-local</Region><RequestId>17E9F3BD14AD3632</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-09 12:05:26,098 ERROR   wandb-upload_0:527183 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/kc1lvzvh/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040517Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=d4f0d45d4b8502d62a750f3f455c6e1257aceaa2c63007ec491f68d1e75ef450: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/kc1lvzvh/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040517Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=d4f0d45d4b8502d62a750f3f455c6e1257aceaa2c63007ec491f68d1e75ef450
2024-08-09 12:05:26,098 ERROR   wandb-upload_0:527183 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '5250', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-09 12:05:26,098 ERROR   wandb-upload_0:527183 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/kc1lvzvh/wandb-metadata.json</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/kc1lvzvh/wandb-metadata.json</Resource><Region>wandb-local</Region><RequestId>17E9F3BD206B65A0</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-09 12:05:27,622 ERROR   wandb-upload_4:527183 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/wandb_artifacts/42/540/8f642e05e18a200ebb54b5ecf2b46c8f?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040519Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=7314ab73bf6efea60d0b531338262d129c41805920572033a03e8d1422de929c: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/wandb_artifacts/42/540/8f642e05e18a200ebb54b5ecf2b46c8f?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040519Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=7314ab73bf6efea60d0b531338262d129c41805920572033a03e8d1422de929c
2024-08-09 12:05:27,622 ERROR   wandb-upload_4:527183 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'x-amz-meta-md5': 'j2QuBeGKIA67VLXs8rRsjw==', 'Content-MD5': 'j2QuBeGKIA67VLXs8rRsjw==', 'Content-Type': 'application/json', 'Content-Length': '25171', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-09 12:05:27,622 ERROR   wandb-upload_4:527183 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>wandb_artifacts/42/540/8f642e05e18a200ebb54b5ecf2b46c8f</Key><BucketName>local-files</BucketName><Resource>/local-files/wandb_artifacts/42/540/8f642e05e18a200ebb54b5ecf2b46c8f</Resource><Region>wandb-local</Region><RequestId>17E9F3BD794FC33C</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-09 12:05:28,423 ERROR   wandb-upload_3:527183 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/wandb_artifacts/42/540/c2df8f0a6cb4e05226b857b6cdc53f4d?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040519Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=161ee69e2e6c5e620bd7fe41cb0f3d56c03e89c5fd96c34cdc728897a14f9c75: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/wandb_artifacts/42/540/c2df8f0a6cb4e05226b857b6cdc53f4d?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040519Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=161ee69e2e6c5e620bd7fe41cb0f3d56c03e89c5fd96c34cdc728897a14f9c75
2024-08-09 12:05:28,423 ERROR   wandb-upload_3:527183 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'x-amz-meta-md5': 'wt+PCmy04FImuFe2zcU/TQ==', 'Content-MD5': 'wt+PCmy04FImuFe2zcU/TQ==', 'Content-Type': 'text/plain; charset=utf-8', 'Content-Length': '5025', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-09 12:05:28,423 ERROR   wandb-upload_3:527183 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>wandb_artifacts/42/540/c2df8f0a6cb4e05226b857b6cdc53f4d</Key><BucketName>local-files</BucketName><Resource>/local-files/wandb_artifacts/42/540/c2df8f0a6cb4e05226b857b6cdc53f4d</Resource><Region>wandb-local</Region><RequestId>17E9F3BDAA86AA96</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-09 12:05:28,579 DEBUG   HandlerThread:527183 [handler.py:handle_request():146] handle_request: keepalive
2024-08-09 12:05:33,581 DEBUG   HandlerThread:527183 [handler.py:handle_request():146] handle_request: keepalive
2024-08-09 12:05:34,631 ERROR   wandb-upload_1:527183 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/kc1lvzvh/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040517Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=e3f48fc3f1005b7250b059ea08238d9d1c0245a1ac1ad9980ea2c8708559a31c: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/kc1lvzvh/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040517Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=e3f48fc3f1005b7250b059ea08238d9d1c0245a1ac1ad9980ea2c8708559a31c
2024-08-09 12:05:34,632 ERROR   wandb-upload_1:527183 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '14196', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-09 12:05:34,632 ERROR   wandb-upload_1:527183 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/kc1lvzvh/code/llama/gmm.py</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/kc1lvzvh/code/llama/gmm.py</Resource><Region>wandb-local</Region><RequestId>17E9F3BF1CF784B5</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-09 12:05:35,148 ERROR   wandb-upload_2:527183 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/kc1lvzvh/diff.patch?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040517Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=89f14b4d3106910c6d563976ed77151506afdab52d67c7f7a996d3977b6a9a17: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/kc1lvzvh/diff.patch?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040517Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=89f14b4d3106910c6d563976ed77151506afdab52d67c7f7a996d3977b6a9a17
2024-08-09 12:05:35,148 ERROR   wandb-upload_2:527183 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '294205', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-09 12:05:35,148 ERROR   wandb-upload_2:527183 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/kc1lvzvh/diff.patch</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/kc1lvzvh/diff.patch</Resource><Region>wandb-local</Region><RequestId>17E9F3BF363005D9</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-09 12:05:35,307 ERROR   wandb-upload_0:527183 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/kc1lvzvh/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040517Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=d4f0d45d4b8502d62a750f3f455c6e1257aceaa2c63007ec491f68d1e75ef450: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/kc1lvzvh/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040517Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=d4f0d45d4b8502d62a750f3f455c6e1257aceaa2c63007ec491f68d1e75ef450
2024-08-09 12:05:35,307 ERROR   wandb-upload_0:527183 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '5250', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-09 12:05:35,307 ERROR   wandb-upload_0:527183 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/kc1lvzvh/wandb-metadata.json</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/kc1lvzvh/wandb-metadata.json</Resource><Region>wandb-local</Region><RequestId>17E9F3BF453CC162</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-09 12:05:35,732 ERROR   wandb-upload_4:527183 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/wandb_artifacts/42/540/8f642e05e18a200ebb54b5ecf2b46c8f?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040519Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=7314ab73bf6efea60d0b531338262d129c41805920572033a03e8d1422de929c: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/wandb_artifacts/42/540/8f642e05e18a200ebb54b5ecf2b46c8f?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040519Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=7314ab73bf6efea60d0b531338262d129c41805920572033a03e8d1422de929c
2024-08-09 12:05:35,733 ERROR   wandb-upload_4:527183 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'x-amz-meta-md5': 'j2QuBeGKIA67VLXs8rRsjw==', 'Content-MD5': 'j2QuBeGKIA67VLXs8rRsjw==', 'Content-Type': 'application/json', 'Content-Length': '25171', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-09 12:05:35,733 ERROR   wandb-upload_4:527183 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>wandb_artifacts/42/540/8f642e05e18a200ebb54b5ecf2b46c8f</Key><BucketName>local-files</BucketName><Resource>/local-files/wandb_artifacts/42/540/8f642e05e18a200ebb54b5ecf2b46c8f</Resource><Region>wandb-local</Region><RequestId>17E9F3BF5D07F599</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-09 12:05:36,878 ERROR   wandb-upload_3:527183 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/wandb_artifacts/42/540/c2df8f0a6cb4e05226b857b6cdc53f4d?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040519Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=161ee69e2e6c5e620bd7fe41cb0f3d56c03e89c5fd96c34cdc728897a14f9c75: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/wandb_artifacts/42/540/c2df8f0a6cb4e05226b857b6cdc53f4d?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040519Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=161ee69e2e6c5e620bd7fe41cb0f3d56c03e89c5fd96c34cdc728897a14f9c75
2024-08-09 12:05:36,878 ERROR   wandb-upload_3:527183 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'x-amz-meta-md5': 'wt+PCmy04FImuFe2zcU/TQ==', 'Content-MD5': 'wt+PCmy04FImuFe2zcU/TQ==', 'Content-Type': 'text/plain; charset=utf-8', 'Content-Length': '5025', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-09 12:05:36,878 ERROR   wandb-upload_3:527183 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>wandb_artifacts/42/540/c2df8f0a6cb4e05226b857b6cdc53f4d</Key><BucketName>local-files</BucketName><Resource>/local-files/wandb_artifacts/42/540/c2df8f0a6cb4e05226b857b6cdc53f4d</Resource><Region>wandb-local</Region><RequestId>17E9F3BFA28E172A</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-09 12:05:38,582 DEBUG   HandlerThread:527183 [handler.py:handle_request():146] handle_request: keepalive
2024-08-09 12:05:43,584 DEBUG   HandlerThread:527183 [handler.py:handle_request():146] handle_request: keepalive
2024-08-09 12:05:48,586 DEBUG   HandlerThread:527183 [handler.py:handle_request():146] handle_request: keepalive
2024-08-09 12:05:52,474 ERROR   wandb-upload_0:527183 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/kc1lvzvh/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040517Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=d4f0d45d4b8502d62a750f3f455c6e1257aceaa2c63007ec491f68d1e75ef450: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/kc1lvzvh/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040517Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=d4f0d45d4b8502d62a750f3f455c6e1257aceaa2c63007ec491f68d1e75ef450
2024-08-09 12:05:52,474 ERROR   wandb-upload_0:527183 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '5250', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-09 12:05:52,474 ERROR   wandb-upload_0:527183 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/kc1lvzvh/wandb-metadata.json</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/kc1lvzvh/wandb-metadata.json</Resource><Region>wandb-local</Region><RequestId>17E9F3C34462B2E8</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-09 12:05:52,534 ERROR   wandb-upload_2:527183 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/kc1lvzvh/diff.patch?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040517Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=89f14b4d3106910c6d563976ed77151506afdab52d67c7f7a996d3977b6a9a17: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/kc1lvzvh/diff.patch?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040517Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=89f14b4d3106910c6d563976ed77151506afdab52d67c7f7a996d3977b6a9a17
2024-08-09 12:05:52,535 ERROR   wandb-upload_2:527183 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '294205', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-09 12:05:52,535 ERROR   wandb-upload_2:527183 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/kc1lvzvh/diff.patch</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/kc1lvzvh/diff.patch</Resource><Region>wandb-local</Region><RequestId>17E9F3C33D080B19</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-09 12:05:53,540 ERROR   wandb-upload_1:527183 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/kc1lvzvh/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040517Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=e3f48fc3f1005b7250b059ea08238d9d1c0245a1ac1ad9980ea2c8708559a31c: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/kc1lvzvh/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040517Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=e3f48fc3f1005b7250b059ea08238d9d1c0245a1ac1ad9980ea2c8708559a31c
2024-08-09 12:05:53,541 ERROR   wandb-upload_1:527183 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '14196', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-09 12:05:53,541 ERROR   wandb-upload_1:527183 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/kc1lvzvh/code/llama/gmm.py</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/kc1lvzvh/code/llama/gmm.py</Resource><Region>wandb-local</Region><RequestId>17E9F3C384099B8E</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-09 12:05:53,588 DEBUG   HandlerThread:527183 [handler.py:handle_request():146] handle_request: keepalive
2024-08-09 12:05:53,765 ERROR   wandb-upload_3:527183 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/wandb_artifacts/42/540/c2df8f0a6cb4e05226b857b6cdc53f4d?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040519Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=161ee69e2e6c5e620bd7fe41cb0f3d56c03e89c5fd96c34cdc728897a14f9c75: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/wandb_artifacts/42/540/c2df8f0a6cb4e05226b857b6cdc53f4d?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040519Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=161ee69e2e6c5e620bd7fe41cb0f3d56c03e89c5fd96c34cdc728897a14f9c75
2024-08-09 12:05:53,765 ERROR   wandb-upload_3:527183 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'x-amz-meta-md5': 'wt+PCmy04FImuFe2zcU/TQ==', 'Content-MD5': 'wt+PCmy04FImuFe2zcU/TQ==', 'Content-Type': 'text/plain; charset=utf-8', 'Content-Length': '5025', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-09 12:05:53,765 ERROR   wandb-upload_3:527183 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>wandb_artifacts/42/540/c2df8f0a6cb4e05226b857b6cdc53f4d</Key><BucketName>local-files</BucketName><Resource>/local-files/wandb_artifacts/42/540/c2df8f0a6cb4e05226b857b6cdc53f4d</Resource><Region>wandb-local</Region><RequestId>17E9F3C3915AF1CA</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-09 12:05:55,259 ERROR   wandb-upload_4:527183 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/wandb_artifacts/42/540/8f642e05e18a200ebb54b5ecf2b46c8f?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040519Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=7314ab73bf6efea60d0b531338262d129c41805920572033a03e8d1422de929c: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/wandb_artifacts/42/540/8f642e05e18a200ebb54b5ecf2b46c8f?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040519Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=7314ab73bf6efea60d0b531338262d129c41805920572033a03e8d1422de929c
2024-08-09 12:05:55,259 ERROR   wandb-upload_4:527183 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'x-amz-meta-md5': 'j2QuBeGKIA67VLXs8rRsjw==', 'Content-MD5': 'j2QuBeGKIA67VLXs8rRsjw==', 'Content-Type': 'application/json', 'Content-Length': '25171', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-09 12:05:55,259 ERROR   wandb-upload_4:527183 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>wandb_artifacts/42/540/8f642e05e18a200ebb54b5ecf2b46c8f</Key><BucketName>local-files</BucketName><Resource>/local-files/wandb_artifacts/42/540/8f642e05e18a200ebb54b5ecf2b46c8f</Resource><Region>wandb-local</Region><RequestId>17E9F3C3E64D35B8</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-09 12:05:58,590 DEBUG   HandlerThread:527183 [handler.py:handle_request():146] handle_request: keepalive
2024-08-09 12:06:03,592 DEBUG   HandlerThread:527183 [handler.py:handle_request():146] handle_request: keepalive
2024-08-09 12:06:08,594 DEBUG   HandlerThread:527183 [handler.py:handle_request():146] handle_request: keepalive
2024-08-09 12:06:13,596 DEBUG   HandlerThread:527183 [handler.py:handle_request():146] handle_request: keepalive
2024-08-09 12:06:18,598 DEBUG   HandlerThread:527183 [handler.py:handle_request():146] handle_request: keepalive
2024-08-09 12:06:23,600 DEBUG   HandlerThread:527183 [handler.py:handle_request():146] handle_request: keepalive
2024-08-09 12:06:28,601 DEBUG   HandlerThread:527183 [handler.py:handle_request():146] handle_request: keepalive
2024-08-09 12:06:29,521 ERROR   wandb-upload_2:527183 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/kc1lvzvh/diff.patch?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040517Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=89f14b4d3106910c6d563976ed77151506afdab52d67c7f7a996d3977b6a9a17: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/kc1lvzvh/diff.patch?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040517Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=89f14b4d3106910c6d563976ed77151506afdab52d67c7f7a996d3977b6a9a17
2024-08-09 12:06:29,521 ERROR   wandb-upload_2:527183 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '294205', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-09 12:06:29,522 ERROR   wandb-upload_2:527183 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/kc1lvzvh/diff.patch</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/kc1lvzvh/diff.patch</Resource><Region>wandb-local</Region><RequestId>17E9F3CBD31F55AC</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-09 12:06:29,742 ERROR   wandb-upload_4:527183 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/wandb_artifacts/42/540/8f642e05e18a200ebb54b5ecf2b46c8f?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040519Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=7314ab73bf6efea60d0b531338262d129c41805920572033a03e8d1422de929c: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/wandb_artifacts/42/540/8f642e05e18a200ebb54b5ecf2b46c8f?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040519Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=7314ab73bf6efea60d0b531338262d129c41805920572033a03e8d1422de929c
2024-08-09 12:06:29,742 ERROR   wandb-upload_4:527183 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'x-amz-meta-md5': 'j2QuBeGKIA67VLXs8rRsjw==', 'Content-MD5': 'j2QuBeGKIA67VLXs8rRsjw==', 'Content-Type': 'application/json', 'Content-Length': '25171', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-09 12:06:29,743 ERROR   wandb-upload_4:527183 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>wandb_artifacts/42/540/8f642e05e18a200ebb54b5ecf2b46c8f</Key><BucketName>local-files</BucketName><Resource>/local-files/wandb_artifacts/42/540/8f642e05e18a200ebb54b5ecf2b46c8f</Resource><Region>wandb-local</Region><RequestId>17E9F3CBECF341ED</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-09 12:06:29,902 ERROR   wandb-upload_3:527183 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/wandb_artifacts/42/540/c2df8f0a6cb4e05226b857b6cdc53f4d?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040519Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=161ee69e2e6c5e620bd7fe41cb0f3d56c03e89c5fd96c34cdc728897a14f9c75: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/wandb_artifacts/42/540/c2df8f0a6cb4e05226b857b6cdc53f4d?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040519Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=161ee69e2e6c5e620bd7fe41cb0f3d56c03e89c5fd96c34cdc728897a14f9c75
2024-08-09 12:06:29,902 ERROR   wandb-upload_3:527183 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'x-amz-meta-md5': 'wt+PCmy04FImuFe2zcU/TQ==', 'Content-MD5': 'wt+PCmy04FImuFe2zcU/TQ==', 'Content-Type': 'text/plain; charset=utf-8', 'Content-Length': '5025', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-09 12:06:29,902 ERROR   wandb-upload_3:527183 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>wandb_artifacts/42/540/c2df8f0a6cb4e05226b857b6cdc53f4d</Key><BucketName>local-files</BucketName><Resource>/local-files/wandb_artifacts/42/540/c2df8f0a6cb4e05226b857b6cdc53f4d</Resource><Region>wandb-local</Region><RequestId>17E9F3CBFB4E84C6</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-09 12:06:30,845 ERROR   wandb-upload_0:527183 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/kc1lvzvh/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040517Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=d4f0d45d4b8502d62a750f3f455c6e1257aceaa2c63007ec491f68d1e75ef450: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/kc1lvzvh/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040517Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=d4f0d45d4b8502d62a750f3f455c6e1257aceaa2c63007ec491f68d1e75ef450
2024-08-09 12:06:30,846 ERROR   wandb-upload_0:527183 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '5250', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-09 12:06:30,846 ERROR   wandb-upload_0:527183 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/kc1lvzvh/wandb-metadata.json</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/kc1lvzvh/wandb-metadata.json</Resource><Region>wandb-local</Region><RequestId>17E9F3CC336DFC30</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-09 12:06:32,194 ERROR   wandb-upload_1:527183 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/kc1lvzvh/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040517Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=e3f48fc3f1005b7250b059ea08238d9d1c0245a1ac1ad9980ea2c8708559a31c: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/kc1lvzvh/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240809%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240809T040517Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=e3f48fc3f1005b7250b059ea08238d9d1c0245a1ac1ad9980ea2c8708559a31c
2024-08-09 12:06:32,194 ERROR   wandb-upload_1:527183 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '14196', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-09 12:06:32,194 ERROR   wandb-upload_1:527183 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/kc1lvzvh/code/llama/gmm.py</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/kc1lvzvh/code/llama/gmm.py</Resource><Region>wandb-local</Region><RequestId>17E9F3CC83C16BBF</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-09 12:06:33,603 DEBUG   HandlerThread:527183 [handler.py:handle_request():146] handle_request: keepalive
2024-08-09 12:06:38,605 DEBUG   HandlerThread:527183 [handler.py:handle_request():146] handle_request: keepalive
2024-08-09 12:06:43,607 DEBUG   HandlerThread:527183 [handler.py:handle_request():146] handle_request: keepalive
2024-08-09 12:06:48,608 DEBUG   HandlerThread:527183 [handler.py:handle_request():146] handle_request: keepalive
